### PR TITLE
New pillar drill page

### PR DESCRIPTION
**Pull request by Marcus6275**

### CODE REVIEW

File: docs/Tools_Equipment/Wood_Working/Pillar_Drill.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Pillar Drill
+
+![](beltsander.jpeg "beltsander.jpeg")
```
Comment (conorriches) [inline docs/Tools_Equipment/Wood_Working/Pillar_Drill.md:3]: Just to let you know this contains the old beltsander image

Comment (GregNGM) [reply docs/Tools_Equipment/Wood_Working/Pillar_Drill.md:3]: There seem to be other sander-related things included - pillar drills don't need abrasive belts

**Review by conorriches — APPROVED** [review]
noice

**Review by GregNGM — APPROVED** [review]
LGTM - needs work but is a good start

**Pull request merged**